packages/evm-adapter/package.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "@heliofi/evm-adapter",
+  "version": "0.0.3-beta",
+  "description": "API to interact with Helio Finances program on EVM chains.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "yarn build -w",
+    "lint": "eslint .",
+    "lint:fix": "yarn lint --fix",
+    "test": "yarn run ts-mocha -t 1000000 test/*.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/heliofi/solidity-adapter.git"
+  },
+  "keywords": [
+    "ethereum",
+    "polygon",
+    "crypto",
+    "stream",
+    "payments"
+  ],
+  "license": "AGPL-3.0-or-later",
+  "bugs": {
+    "url": "https://github.com/heliofi/solidity-adapter/issues"
+  },
+  "homepage": "https://github.com/heliofi/solidity-adapter",
+  "dependencies": {
+    "ethers": "^5.7.2"
+  },
+  "author": "Ivan Britvic<ivbritvic@gmail.com>",
+  "devDependencies": {
+    "@ethersproject/abi": "^5.4.7",
+    "@ethersproject/providers": "^5.4.7",
+    "@heliofi/eslint-config-helio": "^0.0.4",
+    "@nomicfoundation/hardhat-chai-matchers": "^1.0.5",
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.6",
+    "@nomicfoundation/hardhat-toolbox": "^2.0.0",
+    "@nomiclabs/hardhat-ethers": "^2.2.0",
+    "@nomiclabs/hardhat-etherscan": "^3.1.1",
+    "@typechain/ethers-v5": "^10.1.0",
+    "@typechain/hardhat": "^6.1.3",
+    "@types/chai": "^4.3.0",
+    "@types/mocha": "^9.1.0",
+    "@typescript-eslint/eslint-plugin": "^5.19.0",
+    "@typescript-eslint/parser": "^5.19.0",
+    "chai": "^4.3.6",
+    "dotenv": "^16.0.3",
+    "eslint": "^8.13.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-airbnb-typescript": "^17.0.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^26.1.4",
+    "eslint-plugin-prettier": "^4.0.0",
+    "hardhat": "^2.12.4",
+    "hardhat-gas-reporter": "^1.0.9",
+    "mocha": "^9.2.0",
+    "prettier": "^2.5.1",
+    "prettier-plugin-solidity": "^1.0.0",
+    "solidity-coverage": "^0.8.2",
+    "ts-mocha": "^9.0.2",
```

Comment:
do you need mocha ? can't we do everything with jest ?

Reply:
We don't need mocha, removed